### DISCUSSION TITLE
blog/docker-guide-for-dotnet-developers

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#26] [General]
# blog/docker-guide-for-dotnet-developers

This Comprehensive Docker Guide for .NET Developers will cover everything you need to know about Docker, from the basics to advanced concepts. Docker has become an indispensable tool in modern software development, streamlining workflows and boosting productivity, whether you are a .NET developer or work with another tech stack. Whether new to Docker or seeking to deepen your understanding, this guide will provide you with the knowledge and skills necessary to use Docker in your projects effectively.

https://codewithmukesh.com/blog/docker-guide-for-dotnet-developers?utm_source=youtube

**Comment by rajkra** [comment]
Hey this is great Mukesh!! Thanks for all this info. Really helpful.

**Comment by Dasayeve** [comment]
Hey Mukesh, I created a .net 8 app using visual studio and created it with docker support, when I click run it works, creates an image with dev tag and everything works great, but if I try to create another image with other name using the docker file that visual studio generates it does not work when I use docker run with the container created.
Also, I only can brownse localhost:[port] if the application is running in visual studio. You know what may happening?

**Comment by anilpalankar** [comment]
Very well explained Mukesh 
Thanks for the great content

**Comment by YaJohny** [comment]
Thank you it was very helpfull
